mava/systems/jax/launcher.py

```diff
@@ -14,18 +14,32 @@
 # limitations under the License.
 
 """General launcher for systems"""
+import copy
 from typing import Any, Dict, List, Union
 
 import launchpad as lp
 import reverb
 
+from mava.core_jax import SystemBuilder
 from mava.utils import lp_utils
 
 
 class NodeType:
     reverb = lp.ReverbNode
     courier = lp.CourierNode
 
+def copy_store(builder: SystemBuilder, multi_process: bool) -> SystemBuilder:
+    if multi_process:
+        # Note: It is unnecessary to copy when Launchpad is used.
```

Comment:
Just a minor question, but do we not always use Launchpad?

Reply:
Nope launchpad only in the case of multi-process. Check `mava/systems/jax/launcher.py `  
```
if multi_process:
      self._program = lp.Program(name=name)
```

Reply:
we only use lp with multi_process
https://github.com/instadeepai/Mava/blob/develop/mava/systems/jax/launcher.py#L59

Reply:
Thanks for the replies. It makes sense, but just to clarify we are still using the launchpad nodes in single process?

Reply:
As far as I see it, no (https://github.com/instadeepai/Mava/blob/develop/mava/systems/jax/launcher.py#L112) but it is misleading that we check the launchpad node type in the next line

Reply:
We are only using the Launchpad node type to determine if we should create a Reverb server or not in the single process case.

Reply:
@AsadJeewa is correct. We do not use any `launchpad` nodes in the single process system